### ISSUE TITLE
Import from YNAB using split debit/credit fields does not work

### ISSUE THREAD

**Issue by jpjonte** [TheAxelander/OpenBudgeteer#72]
YNAB CSV exports contain a debit and a credit field, but both fields contain values. One contains the amount, and the other contains `0.00`. Example:

```
"Account","Flag","Date","Payee","Category Group/Category","Category Group","Category","Memo","Outflow","Inflow","Cleared"
"Account","","27.01.2022","Salary","...","...","...","",0.00,2000.00,"Cleared"
"Account","","26.01.2022","Expense","...","...","...","",100.00,0.00,"Cleared"
```

As far as I can tell your code only checks for empty/null and not for a zero value, so this file format breaks the import.

**Comment by TheAxelander** [comment]
Acknowledged. I can remember I had this in mind but looks like I missed to add this check too.

**Comment by TheAxelander** [comment]
Fixed with 633a422229434182a2a81ac10e0bee73ae897b7b
